src/buildHTML.js

```diff
@@ -1420,16 +1420,18 @@ groupTypes.accent = function(group, options) {
     } else {
         accentBody = stretchy.svgSpan(group, options);
 
-        if (skew > 0) {
-            // Shorten the accent. That will nudge it to the right.
-            const adjSize = "calc(100% - " + (2 * skew) + "em) 100%";
-            accentBody.style.backgroundSize = adjSize;
-        }
-
         accentBody = buildCommon.makeVList([
             {type: "elem", elem: body},
             {type: "elem", elem: accentBody},
         ], "firstBaseline", null, options);
+
+        const styleSpan = accentBody.children[0].children[0].children[1];
+        styleSpan.classes.push("svg-align");  // text-align: left;
```

Comment:
Yes, you have to open it to see the glorious sight of 575 lines of deleted code.